src/pick/__init__.py

```diff
@@ -176,6 +185,7 @@ def pick(
     default_index: int = 0,
     multiselect: bool = False,
     min_selection_count: int = 0,
+    stdscr: Optional[curses.window] = None
```

Comment:
I'd like to keep this consistency with the param of `Picker`, since `pick` is just a thin wrapper for `Picker`

Reply:
That's fair enough. I originally had them the same but stdscr is common in all the docs so went with that externally.  I'll change and push.

Reply:
`screen` and `stdscr` are both ok for me, just need to keep the name consistent in two places.

Reply:
Done